nvm/pmemobj/tuple.py

```diff
@@ -0,0 +1,96 @@
+import collections
+import sys
+
+from .compat import recursive_repr, abc
+from .list import PersistentList
+from _pmem import ffi
+
+TUPLE_POBJPTR_ARRAY_TYPE_NUM = 50
```

Comment:
Ok, so maybe I'll add one more commet. The hash should be stable, in the sense that it should give exactly the same, unique result between different _compilations_, runs, machines (or whatever else you might think of) for the same type. Otherwise you cannot rely on the type number to clearly identify your type (I've typed the word type a lot in the last two sentences 😄). That's the problem with C++, it doesn't guarantee that in the standard. The problem with hardcoding it is, that someone else is free to use that same number, rendering the feature useless.